depthai_nodes/ml/messages/classification.py

```diff
@@ -91,3 +96,28 @@ def top_score(self) -> float:
         @rtype: float
         """
         return self._scores[0]
+
+    def getVisualizationMessage(self) -> dai.ImgAnnotations:
+        """Returns default visualization message for classification.
+
+        The message adds the top five classes and their scores to the right side of the
+        image.
+        """
+        img_annotations = dai.ImgAnnotations()
+        annotation = dai.ImgAnnotation()
+
+        soritng_indexes = np.argsort(self._scores)[::-1]
+        soritng_indexes = soritng_indexes[:5]
```

Comment:
Isnt the sorting already handled in the creation function?

Reply:
yes but just in case I wanted to do it again if the Classfication message is used in some other parser/message_creator